### PR TITLE
doc: update investment cost README for current calculations

### PR DESCRIPTION
**Pull request by danielolsen**
### Purpose
Update the README in the investment cost folder (this got missed in https://github.com/Breakthrough-Energy/PowerSimData/pull/297).

### What the code is doing
No code.

### Time estimate
15 minutes.

### CODE REVIEW

File: powersimdata/design/investment/README.md [hunk 1]
```diff
@@ -75,63 +67,38 @@ Then all costs are summed by technology (so you can ignore the non-renewables/st
...
-- $Dollar-years are inconsistent. Need a function to convert to inflate to proper year.
+#### Potential improvements:
 
 - If we want to have financial information other than the default ATB values, a separate financials module will be useful for CAPEX/other calculations.
 
-- Figure out issue with transformers described above.
-
-- Find correct wind and solar classes (based on wind speed, irradience) to map to ATB costs and ReEDS regional cost multipliers.
\ No newline at end of file
+- Find correct wind and solar classes (based on wind speed, irradience) to map to ATB costs and ReEDS regional cost multipliers.
```
Comment (rouille) [inline powersimdata/design/investment/README.md:104]: irradience --> irradiance

File: powersimdata/design/investment/README.md [hunk 2]
```diff
@@ -75,63 +67,38 @@ Then all costs are summed by technology (so you can ignore the non-renewables/st
...
 
-**write_poly_shapefile**: Using a csv with specified geometry, creates the shapefile for ReEDS wind resource regions: rs.shp. This shp is used in `plant_to_reeds_reg`.
+`plant_to_reeds_reg`: maps plant locations to ReEDS regions. Used in `_calculate_gen_inv_costs`.
 
-**plant_to_reeds_reg**: maps plant locations to ReEDS regions. Used in `_calculate_gen_inv_costs`.
 
+### Sources
 
-### Sources (more specifics in data folder README.md)
+See **ATTRIBUTION.md**.
```
Comment (rouille) [inline powersimdata/design/investment/README.md:98]: you could create a link to the file here

File: powersimdata/design/investment/README.md [hunk 3]
```diff
@@ -75,63 +67,38 @@ Then all costs are summed by technology (so you can ignore the non-renewables/st
...
+`bus_to_neem_reg`: maps bus locations to NEEM regions. Used in `write_bus_neem_map` and (if there are errors in the mapping file produced in `write_bus_neem_map`), this function is also used in `_calculate_ac_inv_costs`.
 
-**write_bus_neem_map**: maps all base_grid bus locations to NEEM regions and produces a csv mapping file: regionsNEEM.shp. This csv is used in `_calculate_ac_inv_costs`.
+`write_bus_neem_map`: maps all base_grid bus locations to NEEM regions and produces a csv mapping file: regionsNEEM.shp. This csv is used in `_calculate_ac_inv_costs`.
 
 
+#### Functions used for generation/storage regional multiplier mapping
 
-**Functions used for generation/storage regional multiplier mapping**
+`write_poly_shapefile`: Using a csv with specified geometry, creates the shapefile for ReEDS wind resource regions: rs.shp. This shp is used in `plant_to_reeds_reg`.
```
Comment (rouille) [inline powersimdata/design/investment/README.md:91]: Using --> using for consistency

File: powersimdata/design/investment/README.md [hunk 4]
```diff
@@ -75,63 +67,38 @@ Then all costs are summed by technology (so you can ignore the non-renewables/st
...
 
+`sjoin_nearest`: joins a geodataframe of Points and Polygons/Multipolygons. Used in `points_to_polys`.
 
+`points_to_polys`: joins a dataframe (which includes lat and lon columns) with a shapefile. Used in `bus_to_neem_reg` and `plant_to_reeds_reg`.
 
-**Functions used for AC regional multiplier mapping**
+#### Functions used for AC regional multiplier mapping
 
-**bus_to_neem_reg**: maps bus locations to NEEM regions. Used in `write_bus_neem_map` and (if there are errors in the mapping file produced in `write_bus_neem_map`), this function is also used in `_calculate_ac_inv_costs`.
+`bus_to_neem_reg`: maps bus locations to NEEM regions. Used in `write_bus_neem_map` and (if there are errors in the mapping file produced in `write_bus_neem_map`), this function is also used in `_calculate_ac_inv_costs`.
```
Comment (rouille) [inline powersimdata/design/investment/README.md:84]: Not sure I understand the above sentence. I read taht the `bus_to_neem_reg` function is used in `_calculate_ac_inv_costs` only if there are errors in the mapping file.

File: powersimdata/design/investment/README.md [hunk 5]
```diff
@@ -1,72 +1,64 @@
...
-There are 3 separate cost calculations: one for dclines, one for branches, one for plants.
-
+There are 3 separate cost calculations: one for dclines, one for branches (AC transmission), one for plants.
 
-**branches: calculate_ac_inv_costs**
+#### branches: calculate_ac_inv_costs
 
 For all capacity added on a **line**, the investment cost is:
-`Cost ($2010) = rateA (MW) * lengthMi (miles) * costMWmi ($2010/MW-mi) * mult (regional cost multiplier, unitless)`
+`Cost ($today) = rateA (MW) * lengthMi (miles) * costMWmi ($2010/MW-mi) * mult (regional cost multiplier) * inflation(2010 to today)`
```
Comment (rouille) [inline powersimdata/design/investment/README.md:12]: should we write inflation($2010 to today)?

File: powersimdata/design/investment/README.md (hunk 4 above)
Comment (danielolsen) [reply powersimdata/design/investment/README.md:84]: The wording is a bit strange. It is used within `write_bus_neem_map` to create a mapping file, and if a cached mapping file does not map all buses successfully at runtime then it is used as a part of `_calculate_ac_inv_costs`.

File: powersimdata/design/investment/README.md (hunk 1 above)
Comment (danielolsen) [reply powersimdata/design/investment/README.md]: Done.

File: powersimdata/design/investment/README.md (hunk 2 above)
Comment (danielolsen) [reply powersimdata/design/investment/README.md]: Done.

File: powersimdata/design/investment/README.md [hunk 6]
```diff
@@ -1,72 +1,64 @@
...
 Then all lines are summed.
 
 For all capacity added on a **transformer**, the investment cost is:
-`Cost (per transformer at a given kV level, $2010)`
-
-Now that I'm thinking about this again, this is a flawed method because if we're only slightly scaling a transformer, this is overestimating the cost. Might be better to apply to completely new transformers only? Either ignore this cost or reevaluate its calculation.
+`Cost ($today) = rateA (MW) * perMWcost ($2020/MW) * mult (regional cost multiplier) * inflation(2020 to today)`
```
Comment (rouille) [inline powersimdata/design/investment/README.md:16]: same comment about inflation

File: powersimdata/design/investment/README.md (hunk 3 above)
Comment (danielolsen) [reply powersimdata/design/investment/README.md]: Done.

File: powersimdata/design/investment/README.md [hunk 7]
```diff
@@ -1,72 +1,64 @@
...
 
-
-
-**dclines: calculate_dc_inv_costs**
+#### dclines: calculate_dc_inv_costs
 
 For all capacity added on a dcline, the investment cost is:
-`Cost ($2015) = Pmax (MW) * lengthMi (miles) * costMWmi ($2015/MW-mi) + dc_term_cost (DC terminal cost $2015)`
+`Cost ($today) = Pmax (MW) * (lengthMi (miles) * costMWmi ($2015/MW-mi) * inflation(2015 to today) + 2 * terminal_cost_per_MW ($2020) * inflation(2020 to today))`
```
Comment (rouille) [inline powersimdata/design/investment/README.md:21]: same comment about inflation

File: powersimdata/design/investment/README.md (hunk 5 above)
Comment (danielolsen) [reply powersimdata/design/investment/README.md]: Done.

File: powersimdata/design/investment/README.md (hunk 6 above)
Comment (danielolsen) [reply powersimdata/design/investment/README.md]: Done.

File: powersimdata/design/investment/README.md (hunk 7 above)
Comment (danielolsen) [reply powersimdata/design/investment/README.md]: Done.

File: powersimdata/design/investment/README.md [hunk 8]
```diff
@@ -1,72 +1,64 @@
...
+- Lines: Inflation is applied to scale from 2010 dollars to present.
 - Lines: final calculations
-- Xfmrs: Find closest kV present in the corresponding cost table using `select_kv` . Labeled "kV_cost." 
-- Xfmrs: Map kV_cost to corresponding transformer cost.
-
-**Warning**: Now that I think about this again, I bet the transformer costs are too high with this method. If we upgrade the rating just slightly, it's going to count this as an entirely new transformer. We could just drop this investment cost altogether, or just include on totally new transformers? Not sure.
+- Xfmrs: Find closest kV present in the corresponding cost table, for each side of the transformer, to get per-MW values.
+- Xfmrs: Multiply per-MW value by upgraded MW.
+- Xfmrs: Regional multiplier is then applied.
+- Xfmrs: Inflation is applied to scale from 2020 dollars to present.
```
Comment (rouille) [inline powersimdata/design/investment/README.md:49]: Is Xfmrs an abbreviation for transformers? If yes I would use the full word

Comment (danielolsen) [reply powersimdata/design/investment/README.md]: Done

File: powersimdata/design/investment/README.md [hunk 9]
```diff
@@ -75,63 +67,38 @@ Then all costs are summed by technology (so you can ignore the non-renewables/st
 - Select (arbitrary) TechDetail (sub-technology) because we can't differentiate between these yet in our plants.
 - Map base costs to plants by "Technology".
 - Map plant location to ReEDS region (for regional cost multipliers).
-- (messy) If a technology is wind or wind_offshore or csp, regions are in rs (wind resource region), so keep rs is kept as the region to map. If technology is another tech, regions are in rb (BA region), so keep rb as the region to map.
-- Map ["Technology","r" (region)] to ReEDS regional capital cost multipliers. Keep (arbitrary) subclasses for renewables.
+- If a technology is wind or wind_offshore or csp, regions are in rs (wind resource region), so keep rs is kept as the region to map. If technology is another tech, regions are in rb (BA region), so keep rb as the region to map.
```
Comment (rouille) [inline powersimdata/design/investment/README.md:70]: csp --> concentrated solar power
so keep rs is kept as the region to map --> so keep rs as the region to map

Comment (danielolsen) [reply powersimdata/design/investment/README.md:70]: Done.

**Review by rouille — APPROVED** [review]
Thanks

**Pull request merged**